lib/Twig/Test/IntegrationTestCase.php

```diff
@@ -37,7 +71,7 @@ public function testLegacyIntegration($file, $message, $condition, $templates, $
         $this->testIntegration($file, $message, $condition, $templates, $exception, $outputs);
     }
 
-    public function getTests($name, $legacyTests = false)
+    public function getUnitTests($name, $legacyTests = false)
```

Comment:
it is an attempted to follow this suggestion: https://github.com/twigphp/Twig/pull/1805#discussion_r38908773